_episodes/05-performance.md

```diff
@@ -276,40 +276,36 @@ as weaknesses when trying to improve your teaching.  It's good to know what you
 
 ## Feedback
 
-Sometimes it can be hard to receive negative feedback.
+Sometimes it can be hard to receive feedback, especially negative feedback.
 
 ![Feedback Feelings]({{ site.root }}/fig/04/deathbulge-jerk.jpg "Comic from http://www.deathbulge.com/comics/155")
 
-Here is a list of some strategies to make sure that you can receive feedback
+Feedback is most effective when the people involved 
+can share ground rules and expectations.  Here is a list of different 
+ways that you, as the instructor, can set the stage for receiving feedback
 in a way that helps you improve:
 
-* Mental habits matter!  Tell yourself the right things, like:
-    * "It's not personal."
-    * "Look at the positives along with the negatives."
-	* etc.
-
 * Initiate feedback.  It's better to ask for feedback than to receive it unwillingly.
 
-* Choose your own questions.  I'm a big fan of feedback questions like "what is
-  one thing I could have done as a teacher to make this lesson more
-  effective?"  Or "If you could pick one thing from the lesson to go over
-  again, what would it be?"  These questions (hopefully)
-  reveal what didn't work in your teaching, but read as professional suggestions
-  rather than personal judgments.
-
-* Balance positive and negative feedback:
-    * ask for or give "compliment sandwiches" (one positive, one negative, one positive)
-    * ask for both types of feedback
+* Choose your own questions.  For example: 
```

Comment:
* Choose your own questions, and ask for more specific feedback when people just say "that was great". For example: 
   * what was it that you really liked about my teaching?